test/testUtils/FrontEndTestHelper.scala

```diff
@@ -729,6 +734,35 @@ trait FrontEndTestHelper extends PlaySpec with MockitoSugar with Injecting with
     "Failed"
   )
 
+  val standardConsignmentStatusValuesThatGenerateActionLinks: Map[String, List[Option[String]]] =
+    ListMap(
+      // None = user has not started transfer (go to series page)
```

Comment:
I wouldn't add all these comments in here. Comments have a tendency not to get updated, and become incorrect over time, causing more confusion than they actually solve.

This links with my previous comment of writing specific unit tests for the `convertStandardStatusesToUserActions` function. By having these it makes clear what the expected behaviours are, without the need for the comments.